src/quo2/foundations/elevation.cljs

```diff
@@ -0,0 +1,70 @@
+(ns quo2.foundations.elevation
```

Comment:
is it names elevation in figma? not shadow?

Reply:
yeah, I will ask the designers to adjust this. and then update it. shadows is better!